tests/migrations/test_commands.py

```diff
@@ -913,15 +913,16 @@ def test_makemigration_merge_dry_run_verbosity_3(self):
             )
             merge_file = os.path.join(migration_dir, '0003_merge.py')
             self.assertFalse(os.path.exists(merge_file))
-        output = force_text(out.getvalue())
+        output = out.getvalue()
         self.assertIn("Merging migrations", output)
         self.assertIn("Branch 0002_second", output)
         self.assertIn("Branch 0002_conflicting_second", output)
         self.assertNotIn("Created new merge migration", output)
 
         # Additional output caused by verbosity 3
         # The complete merge migration file that would be written
-        self.assertIn("# -*- coding: utf-8 -*-", output)
+        self.assertNotIn("\nb# -*- coding: utf-8 -*-", output)
```

Comment:
I don't think this `\nb#` is actually correct. There should be double quotes after the `b` before `#`.